src/main/java/org/opencds/cqf/tooling/common/CqfmSoftwareSystem.java

```diff
@@ -8,8 +8,20 @@
     protected String version;
     public String getVersion() { return this.version; }
 
-    public CqfmSoftwareSystem(String name, String version) {
+    protected String manufacturer;
+    public String getManufacturer() { return this.manufacturer; }
+
+    public CqfmSoftwareSystem(String name, String version, String manufacturer) {
         this.name = name;
         this.version = version;
+        this.manufacturer = manufacturer;
+
+        if (this.version == null) {
+            this.version = "1.3.1-SNAPSHOT";
```

Comment:
Published jars of cqf-tooling are stamped with the version:
String version = CqfmSoftwareSystem.class.getPackage().getSpecificationVersion();

Reply:
Agreed, corrected